rocky/rocky/scheduler.py

```diff
@@ -199,6 +204,10 @@ def get_task_details(self, task_id) -> Task:
 
     def push_task(self, queue_name: str, prioritized_item: QueuePrioritizedItem) -> None:
         res = self.session.post(f"{self._base_uri}/queues/{queue_name}/push", data=prioritized_item.json())
+
+        if res.status_code == HTTPStatus.BAD_REQUEST and res.json().get("detail") == "not allowed":
```

Comment:
This should work for now, but ideally, we should implement proper error messaging on the scheduler side. (It's also debatable whether this is a client error according to `400 Bad Request` response conventions, but I don't have a better suggestion (yet) and it's not relevant for now)

Reply:
Agreed, right now the 400 also still encapsulates several other possible issues with the request so this might not be a trivial change at this point

Reply:
Please create a new ticket for this work in the scheduler.

Reply:
> Please create a new ticket for this work in the scheduler.

https://github.com/minvws/nl-kat-coordination/issues/1700